docs/guides/vtex-io/Storefront-Guides/pages/vtex-io-documentation-best-practices-for-associating-a-custom-page-with-a-content-type.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Best practices for associating a custom page with a content type"
+slug: "vtex-io-documentation-best-practices-for-associating-a-custom-page-with-a-content-type"
+hidden: false
+createdAt: "2024-07-25T19:50:31.469Z"
+updatedAt: "2024-08-30T17:48:35.094Z"
+excerpt: "Configure URLs effectively when associating custom pages with content types."
+---
+
+[Custom landing pages](https://developers.vtex.com/docs/guides/vtex-io-documentation-creating-a-new-custom-page) are usually associated with specific store URLs. However, in some situations, it is helpful to associate a page template with a **content type** instead. Content types indicate the purpose of a page, such as Product Listing Pages (PLPs) and Product Detail Pages (PDPs).
+
+Creating your own content types can simplify website content management. For example, you can create a content type named "Store Finder" for the web pages that display the cities where your store is located. To associate the assets of these pages with the Store Finder content type, you must set an ID value for Store Finder and use it as a variable in your route paths.
+
+Follow the guidelines below to manage your website content more efficiently and ensure that your assets remain correctly associated with them, even when URLs change.
+
+## Before you begin
+
+Ensure you have created a new page following the guide [Creating a new custom page](https://developers.vtex.com/docs/guides/vtex-io-documentation-creating-a-new-custom-page).
+
+## URL guidelines
+
+To associate a custom landing page with a content type, note the following when creating your URL:
+
+### Setting up content type parameters
+
+- Any URL parameter named `{entity}_id` is identified as a content type. For example, `finder_id` creates the `${finder}` content type.
+
+  > ⚠️ For the URL definition, adding the content type identifier is not enough. You must also attach `_id` next to it. For example, if you create a URL like `/foo/:bar`, the content will be tied to that URL since there is no parameter that can be interpreted as a content type. Thus, if you access _Storefront > Site Editor_ and update the content for `/foo/skywalker`, the content related to any page matching `/foo/:bar` will also be updated.
+
+- The page URL can have only one parameter to indicate a content type.
+- The page URL can have only one parameter ending with `_id`.
+- Any additional variable parameters in the URL are ignored when matching content. For example, if you create the route `/foo/:bar/:{entity}_id`, the same content will be delivered for `/foo/skywalker/1` and `/foo/palpatine/1` as they both relate to the `{$entity}` content type.
```

Comment:
Not sure I understand this part. I love the Star Wars references ⭐🚀, but can you give more realistic examples?